repos/system_upgrade/el7toel8/models/opensshconfig.py

```diff
@@ -36,3 +36,5 @@ class OpenSshConfig(Model):
     """ Value of the MACs directive, if present. """
     modified = fields.Boolean(default=False)
     """ True if the configuration file was modified. """
+    deprecated_directives = fields.List(fields.String())
```

Comment:
btw, you could set the default, so you will not have to change sooo many places just setting the empty list:
```suggestion
    deprecated_directives = fields.List(fields.String(), default=[])
```

Reply:
but you do not have to remove the changes you made - it's ok to keep them. no need to waste a time by manual removal of all these lines - they are valid anyway. I would keep them as they are.

Reply:
Thanks for the tip, will definitely remember this, as it was tedious to perform all the modifications.

The changes already performed has been kept intact.